### ISSUE TITLE
hover accessible view doesn't contain hover contents 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#213931]
found in https://github.com/microsoft/vscode/issues/213662

1. focus a variable in a TS document
2. `cmd/ctrl+k+i` to focus the hover in screen reader mode
3. `alt+f2` to open the accessible view
4. 🐛 see this message, without the hover contents in the accessible view
<img width="603" alt="Screenshot 2024-05-30 at 10 31 41 AM" src="https://github.com/microsoft/vscode/assets/29464607/f02151fc-0894-465f-98a4-dc4904e00adf">

**Comment by aiday-mar** [comment]
Hi I think the PR does not work as intended. Consider the following example, when you focus on the hover and show the accessible view, it show the following content:

'someVar' is declared but its value is never read.ts(6133)const someVar: "{some text}" Quick Fix... (⌘.)

That's because we fetch the innerText, which does not add the new lines between the contents, so the text is on a single line. Furthermore we do not want to necessarily show the `Quick Fix... (⌘.)` marker text. Instead we should show some more explicit text like `There is a quick fix in the hover`. 

This was the reason that when the full content hover is focused, a generic message was shown. To see actual markdown contents, you needed to focus on a specific subpart.

https://github.com/microsoft/vscode/assets/61460952/7b4415d1-64d0-4bbf-83df-e2a824010d56

I think to fix this, we need to implement a method `getFormattedWidgetContent()` on the hover controller which would iterate over all the hover parts and fetch the formatted string for that specific hover part, instead of fetching the innerText directly. 

This PR is already in main and the change I propose will require more code than a simple line. Hence I propose we revert this change and work on it later.

**Comment by meganrogge** [comment]
Hi Aiday, thank you for testing. I think the fix you describe can wait until June. 

IMO, while my PR means there's extra content, poorly formatted, in the accessible view, it fixes the bigger issue - having no content at all.

**Comment by aiday-mar** [comment]
Sounds good, I will look into a fix for June
